### DISCUSSION TITLE
請問deepl pro 翻譯，目標文字可以設為繁體中文嗎

### DISCUSSION THREAD

**Discussion by kehaochehng** [bookfere/Ebook-Translator-Calibre-Plugin#378] [Q&A]
我用deepl pro翻譯引擎，目標文字僅能使用簡體中文，是否可以增加繁體中文

**Comment by bookfere** [comment]
已更新 DeepL 支持语言，请下载最新 [Rolling Release](https://translator.bookfere.com/)  试用。 edce151deac1b66960b074cb85913fef8a6509ac
